go-controller/pkg/util/node_annotations.go

```diff
@@ -86,6 +86,12 @@ const (
 	// ovnkube-node gets the node's zone from the OVN Southbound database.
 	OvnNodeZoneName = "k8s.ovn.org/zone-name"
 
+	// OvnNodeEncapIp is the encap-ip belonging to each local/remote chassis. This is used to
+	// make sure that the chassis handler correctly gets the encap-ip set by the "--encap-ip"
+	// flag. This is especially important in the DPU case where ovn-k runs on behalf of another
+	// cluster.
+	OvnNodeEncapIp = "k8s.ovn.org/encap-ip"
```

Comment:
nit: we should not change capitalization for acronyms. This should be OVNNodeEncapIP and other functions should be like "SetNodeEncapIP". I know you were following the previous pattern so I wont block this PR. But something we should clean up and keep in mind for the future:

https://github.com/golang/go/wiki/CodeReviewComments#initialisms